src/OpenApi/Model/Parameter.php

```diff
@@ -83,7 +83,7 @@ public function getDeprecated(): bool
         return $this->deprecated;
     }
 
-    public function canAllowEmptyValue(): bool
+    public function getAllowEmptyValue(): bool
```

Comment:
BC break? Did you intend to duplicate (like the other ones below) rather than rename the existing?